op-batcher/batcher/driver.go

```diff
@@ -449,9 +453,14 @@ func (l *BatchSubmitter) publishTxToL1(ctx context.Context, queue *txmgr.Queue[t
 		return err
 	}
 
-	if err = l.sendTransaction(ctx, txdata, queue, receiptsCh); err != nil {
-		return fmt.Errorf("BatchSubmitter.sendTransaction failed: %w", err)
-	}
+	waitGroup.Add(1)
```

Comment:
would replace this by a `daQueue.send()`, and again, hide away details of how you manage this